MobileWallet/TariLib/Tor/OnionManager.swift

```diff
@@ -195,53 +195,12 @@ public class OnionManager: NSObject {
                 })
             }
         }
-
         // Wait long enough for tor itself to have started. It's OK to wait for this
         // because Tor is already trying to connect; this is just the part that polls for
         // progress.
         DispatchQueue.main.asyncAfter(deadline: .now() + 1, execute: { [weak self] in
             guard let self = self else { return }
 
-//            if OnionManager.isTorLogging {
```

Comment:
This code is actually something I want to keep trying to add. It fails on travis but I'm checking it every time the tor lib is updated.

Reply:
`DispatchQueue.main.asyncAfter(deadline: .now() + 1,...` does not guarantee that active TorThread finish all tasks.